src/Registration/RegistrationFromTgApp.php

```diff
@@ -248,7 +248,6 @@ private function performLoginWorkFlow(callable $cb): void
             new get_pinned_dialogs(),
             new get_state(),
             new get_app_config(),
-            new get_blocked_contacts(),
```

Comment:
Remove unused class?

Reply:
Yes
New telegram version don't send the request